bin/slurm-send-mail.py

```diff
@@ -499,6 +499,11 @@ def process_spool_file(json_file: pathlib.Path):
     if array_summary or len(jobs) == 1:
         jobs = [jobs[0]]
 
+    if len(jobs) > array_max_notifications:
+        logging.info("Asked to send notifications for %d array-jobs, which exceeds the limit %d. "
+            + "Will send only the first one.", len(jobs), array_max_notifications)
+        jobs = [jobs[0]]
+
```

Comment:
My original idea was to send a summary instead of "too many" singular array-task notifications. And you're right, probably my chosen name for the configuration option is not appropriate. 

However, the change you proposed still solves my issue (too many mails at once), so I appreciate that.

Reply:
I can merge your pull request as it is and adjust the code as above or I can let you update this pull request if you like?